spec/factories/application.rb

```diff
@@ -5,6 +5,7 @@
       student0_application_coding_level: 2,
       student1_application_coding_level: 2,
       student_name: FFaker::Name.name,
+      work_week_ids: [create(:work_week).id],
```

Comment:
Let's keep the factories minimal: all that's needed to have a valid application. So let's just skip the `work_weeks` here, they're not mandatory.